code/game/objects/items.dm

```diff
@@ -395,10 +397,10 @@ var/global/image/fire_overlay = image("icon" = 'icons/effects/fire.dmi', "icon_s
 	M.eye_blurry += rand(3,4)
 	M.eye_stat += rand(2,4)
 	if (M.eye_stat >= 10)
-		M.eye_blurry += 15+(0.1*M.eye_blurry)
-		M.disabilities |= NEARSIGHT
-		if(M.stat != 2)

```

Comment:
This is actually the first time I've touched the code; where can I find the definition of M.stat? Looks like `mob/living/carbon`, but from there I'm lost.

You're probably right, I just want to read the rest of the code first.